src/UnitTests/GitHub.VisualStudio/AssemblyResolverPackageTests.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using GitHub.VisualStudio;
+using Microsoft.VisualStudio.Shell;
+using System.Reflection;
+using System.IO;
+using Xunit;
+
+namespace UnitTests.GitHub.VisualStudio
+{
+    public class AssemblyResolverPackageTests
+    {
+        class TheResolveDependentAssemblyMethod
```

Comment:
These tests aren't being run under ncrunch because this class needs to be `public`.